### DISCUSSION TITLE
Linking consecutive S1 images

### DISCUSSION THREAD

**Discussion by svenborgstrom** [isce-framework/isce2#658] [Q&A] [answered]
Sorry, probably a trivial question.
How can I process two consecutive S1 (IW) images together (obviously same date, orbit, etc.)?
I mean not a small area between two scenes but linking together the whole swaths 1, 2 and 3 of both.
I assume this should be set in "reference.xml" and "secondary.xml" but I don't know how.
Any help will be appreciated, thanks.
Sven.

**Comment by EJFielding** [comment, marked as answer]
Hi Sven,

Maybe you already figured this out, but you can put a list of Sentinel-1 IW slices into the file name part of the input "reference.xml" and "secondary.xml" files. You have to put them in like `["slice1.zip","slice2.zip"]` but then `topsApp.py` will figure out how to get the bursts from two or more slices for the overlap between the two dates.

**Reply by svenborgstrom** [reply]
Hi Eric,
I solved my problem with the indications from
https://github.com/isce-framework/isce2/blob/main/examples/input_files/reference_TOPS_SENTINEL1.xml
B.t.w. thank you for your help.
Cheers,
Sven.
Sven.

**Reply by EJFielding** [reply]
Great to hear you found the answer in the comments of the example input files. Please mark this as answered so others can see it is not open any more.
